src/transforms/sort.js

```diff
@@ -2,6 +2,10 @@ import {ascendingDefined} from "../defined.js";
 import {valueof} from "../mark.js";
 import {basic} from "./basic.js";
 
+export function shuffle(options) {
+  return basic(options, sortValue(Math.random));
+}
```

Comment:
```suggestion
import {basic} from "./basic.js";
import {randomUniform, randomLcg} from "d3";

export function shuffle({seed, ...options}) {
  return basic(options, sortValue(randomUniform.source(randomLcg(seed))()));
}
```

Reply:
maybe
`sortValue((seed ? randomUniform.source(randomLcg(seed)) : randomUniform)())`

Reply:
Okay, added a seed option. We don’t need randomUniform here; we can use Math.random directly.